infrastructure/db/handler.js

```diff
@@ -1,253 +1,208 @@
-* @return {Promise} promise, with json response
+* @return {JSON} Json, that contains all tasks of requested  group
 */
-exports.getGroupTasks = async(groupId) => {
-
-    return new Promise((resolve) => {
-
-        models.group.findByPk(groupId).then((group) => {
-            group.getTasks().then((tasks) => {
-
-                let resp = { tasks : new Array() }
-                for(let i = 0; i < tasks.length; i++) {
-
-                    resp.tasks[i] = {
-                        id : tasks[i].id,
-                        label : tasks[i].label,
-                        status : tasks[i].status,
-                        metadata : tasks[i].metadata
-                    }
-                }
-
-                resolve(resp);
-            });
-        });
-    });
+const getGroupTasks = async(groupId) => {
+    try {
+        return await models.task.findAll({ where : {groupId : groupId}});
+    } catch(err) {
+        throw Error(err);
+    }
 }
 /**
-* Finds all information(groups and tasks and related teammembers) of specific board
-* @param {int} request information of the request
-* @return {Promise} promise, with json response
+* Find board instance by boardId
+* @param {int} boardId id of the wanted board
+* @return {JSON} Json, that contains data of the requested group
 */
-exports.getBoard = async(request) => {
-
-    var response = {};
-    getBoardTeammembers(request.boardId).then((board) => {
-        response.teammembers = board.Teammembers;
-
-        getGroups(request.boardId).then((groups) => {
-
-            response.groups = groups.groups;
-            response.tasks = new Array();
-            for(let i = 0; i < groups.groups.length; i++) {
-                response.tasks[i] = {
-                    groupId : groups.groups[i].id,
-                    tasks : new Array()
-                }
-                //response.tasks[i] = new Array();
-                let groupId = groups.groups[i].id;
-                getGroupTasks(groupId).then((tasks) => {
-
-                    console.log(tasks.tasks);
-                    response.tasks[i].tasks = tasks.tasks.slice();
-                    //response.tasks.tasks = tasks.tasks;
-                    //response.tasks.push(tasks);
-                    console.log(response);
-
-                });
-            }
-        });
-    });
-}
-
-exports.addTask = async(request) => {
-    
-    models.task.create({ label : request.label, status : 'unfinished', 
-        metadata : request.metadata, teammemeberId : request.teammemberId,
-        groupId : request.groupId });
+const getBoard = async(boardId) => {
+    try {
+        return await models.board.findByPk(boardId);
+    } catch(err) {
+        throw Error(err);
+    }
+};
+/**
+* Add task to a specific group, with a teammemberId
+* @param {JSON} task , json with values of the needed fields
+* @return Nothing
+*/
+const addTask = async(task) => {
+    try {
+        models.task.create({ label : task.label, status : 'unfinished', 
```

Comment:
because you already have an input object (`task`) you can use it inside your `models.task.create` method, like
`models.task.create(task)` and maybe inject the `status` field to the object itself? I think it will be more clear. 
(+missing await statement here also)